synth.py

```diff
@@ -52,62 +56,25 @@
     library = gapic.py_library(
         service="firestore_admin",
         version=version,
-        # bazel_target=f"//google/firestore/admin/{version}:firestore-admin-{version}-py",
+        bazel_target=f"//google/firestore/admin/{version}:firestore-admin-{version}-py",
         # include_protos=True,
-        proto_path=f"google/firestore/admin/{version}",
-    )
-    s.move(library / f"google/firestore/admin_{version}", f"google/cloud/firestore_admin_{version}")
-    s.move(library / "tests")
-    s.move(library / "scripts")
-
-    s.replace(
-        f"google/cloud/**/*.py",
-        f"google.firestore.admin_v1",
-        f"google.cloud.firestore_admin_v1",
-    )
-    s.replace(
-        f"tests/unit/gapic/**/*.py",
-        f"google.firestore.admin_v1",
-        f"google.cloud.firestore_admin_v1",
+        # proto_path=f"google/firestore/admin/{version}",
     )
+    s.move(
+        library / f"google/cloud/admin_{version}",
+        f"google/cloud/firestore_admin_{version}",
+         excludes=[
+            library / f"/google/cloud/admin_{version}/__init__.py"
+        ])
+    s.move(library / f"tests", f"tests")
```

Comment:
Do we not need to move the `scripts`?

Reply:
Unsure of it, but I can bring them back in.